src/Compilers/CSharp/Portable/Parser/LanguageParser_InterpolatedString.cs

```diff
@@ -56,8 +56,11 @@ private ExpressionSyntax ParseInterpolatedStringToken()
 
             var originalToken = this.EatToken();
             var originalText = originalToken.ValueText; // this is actually the source text
-            Debug.Assert(originalText[0] == '$');
-            var isVerbatim = originalText.Length > 2 && originalText[1] == '@';
+            Debug.Assert(originalText[0] == '$' || originalText[0] == '@');
+
+            var isVerbatimInterpolated = originalText.Length > 2 && originalText[0] == '@'; // @S
```

Comment:
```@S```?  Did you mean ```@$```?

Reply:
Doesn't appear to be resolved.

---
In reply to: [200850112](https://github.com/dotnet/roslyn/pull/28355#discussion_r200850112) [](ancestors = 200850112)